pepys_import/file/file_processor.py

```diff
@@ -319,12 +306,51 @@ def process_file(self, file_object, current_path, data_store, processed_ctr, imp
 
             # If all tests pass for all parsers, commit datafile
             if not errors:
-                log = datafile.commit(data_store, change.change_id)
-
                 # Keep track of some details for the import summary
                 summary_details = {}
                 summary_details["filename"] = basename
 
+                log = datafile.commit(data_store, change.change_id)
+                metadata_summaries_after = data_store.get_status(
+                    report_metadata=True, exclude=exclude
+                )
+                measurement_summaries_after = data_store.get_status(
+                    report_measurement=True, exclude=exclude
+                )
+                print(
+                    metadata_summaries_after.show_delta_of_rows_added_metadata(
+                        metadata_summaries_before, title="METADATA REPORT"
+                    )
+                )
+                print(
+                    measurement_summaries_after.show_delta_of_rows_added(
+                        measurement_summaries_before, title="MEASUREMENT REPORT"
+                    )
+                )
+                if isinstance(data_store.missing_data_resolver, CommandLineResolver):
+                    choices = (
+                        "Import metadata",
+                        "Import metadata and measurements",
+                        "Don't import data from this file.",
+                    )
+                    choice = self._ask_user_for_finalizing_import(choices)
+                else:  # default is Import metadata and measurements
+                    choice = "2"
+
+                if choice == "1":  # Import metadata
+                    self._remove_measurement(data_store, datafile, change.change_id)
```

Comment:
I think this should be renamed to `_remove_measurements` (ie. plural) as confused me a bit when first reading it.

Reply:
Yes, you're right. It is a typo. I fixed it: https://github.com/debrief/pepys-import/pull/611/commits/a2c585029376e496c8055ae19fe95ffefbc9f7e0